src/sentry/static/sentry/app/api.tsx

```diff
@@ -96,6 +143,7 @@ export class Client {
       if (req && req.alive) {
         // Check if API response is a 302 -- means project slug was renamed and user
         // needs to be redirected
+        // @ts-ignore
         if (this.hasProjectBeenRenamed(...args)) {
           return;
```

Comment:
`this. hasProjectBeenRenamed()` expects `JQueryXHR` as the input; but this cannot be guaranteed.

Introduced in https://github.com/getsentry/sentry/pull/8799